### PR TITLE
Attempt to resolve potential overlay crashes at shutdown

### PR DESCRIPTION
**Pull request by jherico**
## Testing

Enter a domain with streaming video or audio in a web entity.  Shutdown the application.  Should not crash.

### CODE REVIEW

**Comment by hifi-gustavo** [comment]
Android build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/8315/app_12726_PR.apk

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/8315/HighFidelity-Beta-PR12726-29106fbbbe422527fed7b5f10a6f57bea2528124.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/8315/HighFidelity-Beta-PR12726-29106fbbbe422527fed7b5f10a6f57bea2528124.exe

**Comment by hifi-gustavo** [comment]
Android build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/8316/app_12726_PR.apk

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/8316/HighFidelity-Beta-PR12726-9be162de95293d911cbaddfb1e772c53e4599ddf.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/8316/HighFidelity-Beta-PR12726-9be162de95293d911cbaddfb1e772c53e4599ddf.exe

File: interface/src/ui/overlays/QmlOverlay.h [hunk 1]
```diff
@@ -32,10 +32,11 @@ class QmlOverlay : public Overlay2D {
     void render(RenderArgs* args) override;
 
 private:
+    Q_INVOKABLE void qmlElementDestroyed();
     Q_INVOKABLE void buildQmlElement(const QUrl& url);
 
 protected:
-    std::shared_ptr<QQuickItem> _qmlElement;
+    QQuickItem* _qmlElement{ nullptr };
```
Comment (samcake) [inline interface/src/ui/overlays/QmlOverlay.h]: Shouldn' we use a weakptr here ?

Comment (samcake) [reply interface/src/ui/overlays/QmlOverlay.h]: WE can Not ok i get it

**Comment by qlimnavarro** [comment]
Currently Testing

**Comment by qlimnavarro** [comment]
Pass @conklin94122 
Build did not crash when exiting while a video on web entity was playing or audio was streaming.

**Pull request merged**